src/components/main/AuthMain.tsx

```diff
@@ -1,5 +1,34 @@
-import RestClient from '@/components/restClient/RestClient';
+import { Button, Card, Flex, Typography } from 'antd';
+import Link from 'next/link';
+import { useAuthState } from 'react-firebase-hooks/auth';
+
+import { auth } from '@/lib/auth/firebase.config';
+import { useAuth } from '@/lib/auth/useAuth';
+
+const { Title } = Typography;
 
 export default function AuthMain() {
-  return <RestClient />;
+  const [user] = useAuthState(auth);
+  const { logout } = useAuth();
+
+  return (
+    <Flex vertical gap={10} justify="center" align="center" style={{ height: '100vh' }}>
+      <Card>
+        <Title>{`Welcome back, ${user?.displayName || 'user'}`}</Title>

```

Comment:
```suggestion
<Flex vertical gap={10} justify="center" align="center" style={{ height: '100vh' }}>
          <Card
            style={{
              width: '25%',
              minWidth: '255px',
              textAlign: 'center',
              backgroundColor: 'transparent',
              border: 'solid var(--color-additional-light)',
              boxShadow: '0 4px 10px var(--color-additional)',
            }}
          >
            <Card>
              <Title>{`Welcome back, ${user?.displayName || 'user'}`}</Title>

```

Если обернуть твою карточку в мою с неавторизованного мейна то получается вот так
<img width="541" height="305" alt="image" src="https://github.com/user-attachments/assets/beb72ae1-c6f6-4173-8f93-e8d5cb8b6441" />

еще надо посмотреть - обернуть в какой нибудь контейнер с белым bg и закругленными углами, чтоб было консистентно с мейном
<img width="280" height="219" alt="image" src="https://github.com/user-attachments/assets/a28eb707-bc57-4e0a-9ab0-ec2b3d15d074" />
только не на всю страницу как на мейне а просто как широкие поля для карточки
рас у нас тут ничего нет, надо добавить просто слоев-цветов

Reply:
Добавил)